src/consensus/params.h

```diff
@@ -220,6 +289,13 @@ struct Params {
         int endHeight,
         const std::vector<std::string>& addresses);
 
+    /**
+     * A set of features that have been explicitly force-enabled
+     * via the CLI, overriding block-height based decisions for
+     * this feature.
```

Comment:
This doesn't block this PR, but: now that I think about it, it might be useful to be able to test, via a command-line argument, the activation of a feature at a particular height.